src/uct/base/uct_iface.c

```diff
@@ -608,7 +608,7 @@ ucs_config_field_t uct_iface_config_table[] = {
    "Priority of methods to allocate intermediate buffers for communication",
    ucs_offsetof(uct_iface_config_t, alloc_methods), UCS_CONFIG_TYPE_ARRAY(alloc_methods)},
 
-  {"FAILURE", "error",
+  {"FAILURE", "diag",
```

Comment:
we need to check the error status and hide the error only if it's UCS_ERR_ENDPOINT_TIMEOUT (as translated by uct_ib_mlx5_completion_with_err or uct_rc_verbs_wc_to_ucs_status for example). In case of any other error it's considered a local problem, and should not be hidden.

Reply:
so then the default of this param shouldn't change and remain 'error'?

Reply:
need to update error handling routines to use this config param when the CQE indicates connection error. in other cases need to use ucs_fatal